ur5_joint_limits:
    shoulder_pan_joint:
      upper_limits: 3.1415926
      lower_limits: -3.1415926
    shoulder_lift_joint:
      upper_limits: 3.1415926
      lower_limits: -3.1415926
    shoulder_tilt_joint:
      upper_limits: 3.1415926
      lower_limits: -3.1415926
    elbow_joint:
      upper_limits: 3.1415926
      lower_limits: -3.1415926
    wrist_1_joint:
      upper_limits: 3.1415926
      lower_limits: -3.1415926
    wrist_2_joint:
      upper_limits: 3.1415926
      lower_limits: -3.1415926
    wrist_3_joint:
      upper_limits: 3.1415926
      lower_limits: -3.1415926